reviews/api/views.py

```diff
@@ -296,20 +306,11 @@ def get_queryset(self):
         """Returns all proposals that still need to be reviewed by the supervisor"""
         objects = Decision.objects.filter(
             go="",
-            review__stage=Review.Stages.SUPERVISOR,
-            review__proposal__status=Proposal.Statuses.SUBMITTED_TO_SUPERVISOR,
+            review__is_committee_review=False,
```

Comment:
This API endpoint should return *open* supervisor reviews, so we still need to check the stage and/or proposal status. Or figure out if the review is still open some other way.